src/webhook/test/unit/webhook.service.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { AppModule } from '../../../app.module';
+import { RecipeResponse } from '../../../recipe/responses/recipe.response';
+import { WebhookService } from '../../webhook.service';
+import { HttpService } from '@nestjs/axios';
+import { AxiosResponse } from 'axios';
+import { of } from 'rxjs';
+
+const recipeForWebhook: RecipeResponse = {
+  id: 1,
+  name: 'testName',
+  description: 'testDescription',
+  fileId: 1,
+  authorId: 1,
+  preparing: [
+    {
+      step: 'testStep',
+      order: 1,
+    },
+  ],
+  ingredients: [
+    {
+      name: 'testName',
+      amount: 'testAmout',
+    },
+  ],
+  fileUrl: 'testFileUrl',
+};
+
+describe('Webhook Service', () => {
+  let webhookService: WebhookService;
+  let httpService: HttpService;
+
+  beforeAll(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    await module.createNestApplication().init();
+    webhookService = module.get<WebhookService>(WebhookService);
+    httpService = module.get<HttpService>(HttpService);
+  });
+
+  afterEach(async () => {
+    jest.clearAllMocks();
+    jest.restoreAllMocks();
+  });
+
+  it('should send a webhook', async () => {
+    const webhookUrl = process.env.WEBHOOK_URL;
+
+    const response: AxiosResponse<any> = {
+      data: recipeForWebhook,
+      headers: {},
+      config: {
+        url: 'http://localhost:3000/mockUrl',
+        headers: undefined,
+      },
+      status: 200,
+      statusText: 'OK',
+    };
+
+    const httpPost = jest
```

Comment:
This should be moved to the root of the `describe`, in before each in case you would add another test case and forgot about the mock.